src/components/Radio.tsx

```diff
@@ -40,18 +43,38 @@ const StyledInput = styled.input`
   }
 
   &:checked::before {
-    opacity: 1;
+    opacity: ${(props => props.isDisabled ? 0 : 1)};
   }
 `;
 
+const LabelWithTooltipWrapper = styled.div`
+  display: inline-block;
```

Comment:
In Firefox, for some reason this causes like 1px of pixel shift, so combos of radios with and without tooltips will have different spacing? I can't think of any fixes but mentioning in case you have an idea.